catalyst_voices/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   animated_text_kit: ^4.2.2
   animations: ^2.0.11
+  bip39: ^1.0.6
```

Comment:
Please add this dependency in melos.yaml. The reason for this is we have a lot of packages and they must share the same version of this dependency, melos.yaml synchronizes them in all packages.